controllers/subscription/subscription_controller.go

```diff
@@ -269,6 +281,23 @@ func (r *SubscriptionReconciler) HandleUpgrades(ctx context.Context, rhmiSubscri
 	}, nil
 }
 
+func (r *SubscriptionReconciler) upgradeAllowed() (bool, error) {
+	upgradeConfigMap := &corev1.ConfigMap{}
+	err := r.Get(context.TODO(), types.NamespacedName{Namespace: r.operatorNamespace, Name: RHOAMUpgradeConfigMap}, upgradeConfigMap)
+	if err != nil {
+		if !k8serr.IsNotFound(err) {
+			return false, err
+		}
+	}
+
+	if upgradeConfigMap.Data["allowRhoamUpdates"] == "false" {
```

Comment:
Do we need to have any data in it or is it enough that the config map is present?

Reply:
not really that we need, I had a discussion about it with Brian and we both agreed that having an entry in the cm doesn't harm us and brings extra clarity on what is this supposed to do.